dawn/src/dawn/IIR/StencilMetaInformation.cpp

```diff
@@ -200,8 +200,6 @@ void StencilMetaInformation::finalizeStencilFunctionSetup(
     insertExprToStencilFunctionInstantiation(stencilFun);
   }
 
-  stencilFun->update();
```

Comment:
This is not needed, since, when `finalizeStencilFunctionSetup()` is called, the underlying ast (the one of `doMethod_`) is empty. Later on, the ast is set and then `update()` is called.